### ISSUE TITLE
docs: ngsw-config tool?

### ISSUE THREAD

**Issue by pliski** [angular/angular#26624]
## I'm submitting a...

<pre><code>[x] Documentation issue or request
</code></pre>

## Current behavior

In the [Service worker configuration guide](https://angular.io/guide/service-worker-config) there is a path mismatch in the first code block:
`$ ngsw-config dist src/ngsw.json /base/href`  

## Expected behavior
Should be either:
`$ ngsw-config dist ngsw-config.json /base/href`

or:

`$ ngsw-config dist dist/ngsw.json /base/href`

Can't tell as I was not able to find any further information about the ngsw-config tool.     

## Version:
<pre><code>
Angular.io version: 7.0.1-build.51211+sha.c8adef2.

</code></pre>

## Others:

Would be nice to have more info about the ngsw-config tool.

**Comment by gkalpak** [comment]
There's not much to the `ngsw-config` tool, nor is it expected for people to be using it manually (except for very rare usecases).
It just takes the `ngsw-config.json` configuration file and generates a manifest (i.e. an `ngsw.json`), which has to be served along with the rest of the app in order for the SW to...do its thing.

You are right, though; the command is out-of-date as our other paths mentioned in the guide (e.g. `src/ngsw-config.json` is now by default in `ngsw-config.json`).

##
Basically, `ngsw-config` is supposed to be called like this (where `<base-href>` is basically the value you would have passed to `ng build`'s `--baseHref` option):

```
ngsw-config <output-path> <config-path> [<base-href>]
```

For example:

```
# Without `--baseHref`
ngsw-config dist/my-project ngsw-config.json

# With `--baseHref`
ngsw-config dist/my-project ngsw-config.json /my-app/
```

##
@pliski (or anyone else): PRs with docs fixes/improvements are always welcome :wink:

**Comment by rockument69** [comment]
This has been fixed. The command in the doc is now the following:

`./node_modules/.bin/ngsw-config ./dist/<project-name> ./ngsw-config.json [/base/href]`
